docs/errors.md

```diff
@@ -149,46 +195,20 @@ Trailing slash path mappings for packages are a way of mapping any subpath of th
 </script>
 ```
 
-SystemJS Error #6 is logged by SystemJS to implement [Step 6.1 of this part of the import maps spec](https://wicg.github.io/import-maps/#sort-and-normalize-a-specifier-map).
+SystemJS Warning #52 is logged by SystemJS to implement [Step 6.1 of this part of the import maps spec](https://wicg.github.io/import-maps/#sort-and-normalize-a-specifier-map).
 
-## 7
+## 53
 
-### AMD require is not supported
+### Invalid module id
```

Comment:
```suggestion
### ID is not a valid URL to set in the registry
```

I think it would be nice to match the wording of the message to the titles here.